ols/src/cache/postgres_cache.py

```
@@ -166,6 +182,120 @@ def _ddl_statements(self) -> list[str]:
+            cursor.execute(
+                "SET statement_timeout = %s", (str(config.statement_timeout),)
+            )
+
+    def shutdown(self) -> None:
+        """Stop the health-check thread and close its connection."""
+        self._shutdown_event.set()
+        self._health_thread.join(timeout=self._health_check_connect_timeout)
+        if self._health_connection is not None:
+            try:
+                self._health_connection.close()
+            except Exception as close_err:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)